annotation/Pipfile.lock

```diff
@@ -1,12 +1,10 @@
 {
```

Comment:
This file should be in the .gitignore I think?

Reply:
It was added to the gitignore in my branch #112

Reply:
I plan on moving away from Pipfiles in favour of `virtualenv` to stay more vanilla and GCP will handle this for us on the deployment side. For the time being, happy to leave that change in #112.